web/src/types/storage.ts

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Should we explain the differences between `api/storage/types` and `types/storage` somewhere? What is more, should we have a plan for _unification_?

Reply:
Yes, we have a plan for unification, but it will take quite some time.